### DISCUSSION TITLE
Does the volume server operator assume uniform PVs?

### DISCUSSION THREAD

**Discussion by ProjectInitiative** [seaweedfs/seaweedfs-operator#55] [Q&A] [answered]
Doing some testing, it seems as though the only way to scale and add more volume servers is by changing the replica number. It also appears that the operator tries to create PVCs based on PV availability. Given several k8s nodes with varying amount of disks and sizes, the operator appears to not be using the available disks, and simply creating uniform volume servers.

I could be wrong, but I think in the current state the operator is really only meant to deploy uniform servers. Any insight is much appreciated!

**Comment by chrislusf** [comment, marked as answer]
Your observation is correct. This operator was created assuming the system will be running on cloud servers, where new disks will just be requested with a uniform disk size.

**Reply by ProjectInitiative** [reply]
Wonderful, thank you for the clarification!
